spot-ml/src/main/scala/org/apache/spot/SuspiciousConnectsArgumentParser.scala

```diff
@@ -45,6 +46,10 @@ object SuspiciousConnectsArgumentParser {
       action((x, c) => c.copy(topicCount = x.toInt)).
       text("topic count")
 
+    opt[String]("userdomain").required().valueName("<user domain>").
```

Comment:
if it's not required,  then what should it default to?
should we just act like there is no user domain and only separate domains into Top-1M  / Top-1M classes if dns is run with no provided user domain?  

honestly, that would be what I would do